src/cf-core/src/cf-base.less

```diff
@@ -556,14 +556,16 @@ select[multiple] {
     border: 1px solid @input-border;
     border-radius: 0;
     vertical-align: top;
-    -webkit-appearance: none;
+    appearance: none;
+    // TODO: update to use CSS 3 Basic UI specs as user-modify is deprecated.
     -webkit-user-modify: read-write-plaintext-only;
+    -moz-user-modify: read-write-plaintext-only;
```

Comment:
Removed in https://github.com/cfpb/capital-framework/pull/359/commits/6909fc553a1e1c33b7fa05604f5f383a620976df